src/clientAuth/bearerBody.js

```diff
@@ -8,12 +14,11 @@ class BearerBody extends ClientAuthnMethod {
   }
 
   /**
-   * More complicated logic then I would have liked it to be
-   *@param {*} cis Request class instance
-   *@param {*} ci Client information
-   *@param {*} requestArgs Request arguments
-   *@param {*} httpArgs HTTP header arguments
-   *@param {*} kwargs
+   *@param {ResourceRequest} cis Request class instance
```

Comment:
Add a space between * and @param (sorry I missed this earlier).

Reply:
Fixed